ui/partials/main/jobs.html

```diff
@@ -55,12 +55,18 @@
               !isLoadingJobs && locationHasSearchParam('revision') && currentRepo.url"
               class="result-set-body unknown-message-body">
   <span ng-init="revision=getSearchParamValue('revision')">
-    <span>Waiting for a push with revision <strong>{{revision}}</strong></span>
-    <a href="{{currentRepo.getPushLogHref(revision)}}"
-       target="_blank"
-       title="open revision {{revision}} on {{currentRepo.url}}">(view pushlog)</a>
-    <span class="fa fa-spinner fa-pulse th-spinner"></span>
-    <div>If the push exists, it will appear in a few minutes once it has been processed.</div>
+    <span ng-if="revision!=='undefined'">
```

Comment:
We're guilty in a few places in our code base, but it seems the preferred style in angular and js in general is whitespace between the operator for readability. :)

`ng-if="revision !== 'undefined'"`